.eslintrc.js

```diff
@@ -121,7 +121,7 @@ module.exports = {
       ],
       rules: {
         'import/first': 'off',
-        'chameleon/vue-props-interface': 'error',
+        // 'chameleon/vue-props-interface': 'error',
```

Comment:
Now that we can use imported types for props this rule seems to not be necessary anymore. However, since we are using a hacky solution to achieve that I was wondering if we should get rid of the rule completely or it would be better to keep it commented for now?

Reply:
I wouldn't remove this in the current PR as we have another check in it(typing defineProps with generic) and we should check if import is working fine everywhere if we remove interfaces from vue files.

Reply:
I see, then I'm going to revert that and add a `eslint-disable` comment for that rule in this component

Reply:
I'd prefer to use the same way we use elsewhere, and in the separate ticket move all at once on importing props approach.

Reply:
mm... okay! Then should I remove the fix and add it again in that future ticket?

Reply:
yes